tests/conftest.py

```diff
@@ -22,49 +36,62 @@
 )
 
 MASTER_KEY = "masterKey"
-BASE_URL = "http://127.0.0.1:7700"
+BASE_URL = "https://127.0.0.1:7700"
 
 ROOT_PATH = Path().absolute()
 SMALL_MOVIES_PATH = ROOT_PATH / "datasets" / "small_movies.json"
 
+_SSL_VERIFY: ssl.SSLContext | bool = (
+    truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT) if has_truststore else False
+)
```

Comment:
```suggestion
_SSL_VERIFY: ssl.SSLContext | bool = (
    truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT) if truststore else False
)
```

Reply:
i made it diferently ( now there is better generation logic inside fixture